pkg/js/README.md

```diff
@@ -0,0 +1,146 @@
+# OpenFGA Language - JS
+
+[![@openfga/syntax-transformer](https://img.shields.io/npm/v/%40openfga%2Fsyntax-transformer/beta?label=%40openfga%2Fsyntax-transformer&style=flat-square)](https://www.npmjs.com/package/@openfga/syntax-transformer)
```

Comment:
If we can move this below the intro for each README, it will be great as some use this line as a summary.

So have the intro line and the badges after if you can